plugin/src/test/groovy/com/novoda/staticanalysis/internal/detekt/DetektIntegrationTest.groovy

```diff
@@ -25,6 +28,26 @@ class DetektIntegrationTest {
         this.projectRule = projectRule
     }
 
+    @Test
+    void givenOutputNotDefinedShouldFailBuildOnConfiguration() {
```

Comment:
the name of this test could be `shouldFailBuildOnConfigurationWhenOutputNotDefined`

Reply:
haha 😃The test name was like that in the beginning actually.